csharp/ql/src/experimental/ir/implementation/unaliased_ssa/internal/SSAConstruction.qll

```diff
@@ -718,7 +718,7 @@ module DefUse {
   }
 
   /**
-   * Gets the rank index of a hyphothetical use one instruction past the end of
+   * Gets the rank index of a hypothetical use one instruction past the end of
```

Comment:
equivalent to https://github.com/github/codeql/pull/10825/commits/10ec41bc8f2f863321ebd536c6752491208d2d98#diff-f5cf97073ed3726cc84eb446644c71394729d149e48b27d8a26ea7099a821d3bR721